### PR TITLE
Update comments in GHA script

### PR DESCRIPTION
**Pull request by sjspielman**
This PR updates a comment I noticed was stale; the code is correct, though.

### CODE REVIEW

File: .github/workflows/copy-completed-notebooks.yml [hunk 1]
```diff
@@ -52,8 +52,8 @@ jobs:
           # For each module, the directory of notebooks to copy are:
           #  Intro R: intro_to_R_tidyverse/
```
Comment (jashapiro) [inline .github/workflows/copy-completed-notebooks.yml:53]:
```suggestion
          #  Intro R: intro-to-R-tidyverse/
```

File: .github/workflows/copy-completed-notebooks.yml [hunk 2]
```diff
@@ -52,8 +52,8 @@ jobs:
...
           #  Intro scRNA-seq: intro_to_R_tidyverse/ and scRNA-seq/
-          #  Advanced scRNA-seq: advanced-scRNA-seq/
-          #  bulk RNA-seq: RNA-seq/
+          #  Advanced scRNA-seq: scRNA-seq-advanced/
+          #  bulk RNA-seq: intro_to_R_tidyverse/, RNA-seq/, and pathway-analysis/
```
Comment (jashapiro) [inline .github/workflows/copy-completed-notebooks.yml:56]:
```suggestion
          #  bulk RNA-seq: intro-to-R-tidyverse/, RNA-seq/, and pathway-analysis/
```

**Review by jashapiro — APPROVED** [review]
Honestly, I would just take out most of this comment as I don't think it provides any information not in in the code pretty clearly. But if you want to keep it, there are other errors as well as well.

**Comment by sjspielman** [comment]
> Honestly, I would just take out most of this comment as I don't think it provides any information not in in the code pretty clearly. But if you want to keep it, there are other errors as well as well.

I would honestly prefer not to keep it as well - not worth maintaining the comments given the clarity of code.

**Pull request merged**